website_sale_product_reference_displayed/views/website_sale_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Tecnativa - David Vidal
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <template id="products_item" name="Display product reference" inherit_id="website_sale.products_item" customize_show="True">
+        <!-- Product title in list -->
+        <xpath expr="//a[@t-field='product.name']" position="attributes">
+            <attribute name="class" add="d-none" separator=" "/>
+        </xpath>
+        <xpath expr="//a[@t-field='product.name']" position="after">
+            <a itemprop="name" t-att-href="product_href" t-att-content="product.name" t-field="product.name"/>
+        </xpath>
```

Comment:
Wouldn't it be easier to just alter those attributes in the 1st matched element? :thinking:

Reply:
There was a mistake here (solved).

Anyway, the attribute I want to change is the field value itself. So the approach is:

- hide the original field (product.name)
- show a copy of the same tag but with the other (product.display_name)